assets/queries/ansible/aws/db_instance_publicly_accessible/query.rego

```diff
@@ -0,0 +1,59 @@
+package Cx
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    isAnsibleTrue(task["community.aws.rds_instance"].publicly_accessible)
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.{{community.aws.rds_instance}}.publicly_accessible", [task.name]),
+        "issueType": "IncorrectValue",
+        "keyExpectedValue": "community.aws.rds_instance.publicly_accessible should be false",
+        "keyActualValue": "community.aws.rds_instance.publicly_accessible is true"
+    }
+}
+
+CxPolicy[result] {
+    document := input.document[i]
+    tasks := getTasks(document)
+    task := tasks[t]
+    isAnsibleTrue(task.rds.publicly_accessible)
+    result := {
+        "documentId": document.id,
+        "searchKey": sprintf("name=%s.rds.publicly_accessible", [task.name]),
+        "issueType": "IncorrectValue",
+        "keyExpectedValue": "aws.rds.publicly_accessible should be false",
+        "keyActualValue": "aws.rds.publicly_accessible is true"
```

Comment:
The collections Amazon.aws does not seem to have this module https://docs.ansible.com/ansible/latest/collections/amazon/aws/index.html#plugins-in-amazon-aws

Reply:
Removed `aws`
example: https://github.com/AutomationWithAnsible/ansible-aws-rds/blob/master/tasks/rds.yml#L4-L5

Reply:
the module doesn't exist in the docs, the link mentioned in that example is not found (404) in the docs